app/src/main/java/com/github/houseorganizer/houseorganizer/MainActivity.java

```diff
@@ -19,15 +19,14 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     /* Called when the user taps the Send button */
-    @SuppressWarnings("unused")
     public void sendMessage(View view) {
         // Get name
         EditText editText = findViewById(R.id.mainName);
         String message = editText.getText().toString();
 
         // An intent is an object that provides runtime binding between separate components
         // The intent represents an app's intent to do something
-        Intent intent = new Intent(this, GreetingActivity.class);
+        Intent intent = new Intent(this, MainScreenActivity.class);
         // An intent can carry data types as key-value pairs called extras
         intent.putExtra(EXTRA_MESSAGE, message);
```

Comment:
As the MainScreenActivity does not require a message, it should be better to entirely remove the message sending to not be confused later.